### ISSUE TITLE
Empty host and service groups not shown

### ISSUE THREAD

**Issue by gitmopp** [Icinga/icingaweb2#2796]
If I define a hostgroup in Icinga2 then Icinga2 knows the hostgroup and also modules like livestatus will show the hostgroup.
But IcingaWeb2 does not show the group. Is it a bug or feature? From my point of view it should be visible in IcingaWeb2 to have a consistent view through all tools / modules.

Sample configuration:
```
object HostGroup "Test Printer" {
  display_name = "Test Printer"
  assign where host.vars.group == "test-printer"
}
```

```
icinga2 object list --type hostgroup --name "Test Printer"
Object 'Test Printer' of type 'HostGroup':
  % declared in '/etc/icinga2/conf.d/hostgroups/test/test-printers.conf', lines 1:0-1:28
  * __name = "Test Printer"
  * action_url = ""
  * display_name = "Test Printer"
    % = modified in '/etc/icinga2/conf.d/hostgroups/test/test-printers.conf', lines 2:3-2:29
  * groups = null
  * name = "Test Printer"
  * notes = ""
  * notes_url = ""
  * package = "_etc"
  * templates = [ "Test Printer" ]
    % = modified in '/etc/icinga2/conf.d/hostgroups/test/test-printers.conf', lines 1:0-1:28
  * type = "HostGroup"
  * vars = null
  * zone = ""
```

**Comment by leeclemens** [comment]
Can confirm, not sure if it's a bug though. What do you hope to learn from a hostgroup without any members having it's members statuses (not) listed?  Icinga Director will show empty hostgroups and allow you to interact/apply, etc.

From the perspective of keeping the config clean and removing unused/erroneous hostgroups, I could see it being useful having them displayed. But from the perspective of viewing the status of hosts and services, empty groups are irrelevant, imho.

**Comment by gitmopp** [comment]
Uses cases for an empty group, e. g. you have some kind of dashboard or direct link to the hostgroup. On the dashboard you want to see right now there are no members in the group.
For example the hostgroup could include all your AWS EC2 instances, but on the next day you delete all EC2 instances so the group is empty again. This way you could see on the dashboard you have 0 EC2 hosts. Of course there are also other ways to display this.

**Comment by lippserd** [comment]
Hi,

Thanks for the report. We will fix that issue with our 2.5.0 release.

Best regards,
Eric
